src/Compilers/VisualBasic/Portable/PredefinedPreprocessorSymbols.vb

```diff
@@ -1,14 +1,11 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 Imports System.Collections.Immutable
-Imports Microsoft.CodeAnalysis.Text
-Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
-Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Microsoft.CodeAnalysis.VisualBasic
     Public Module PredefinedPreprocessorSymbols
 
-        Friend Const CurrentVersionNumber = 15.3
+        Friend Const CurrentVersionNumber = 15.6
```

Comment:
Here's the specific test: `Microsoft.CodeAnalysis.VisualBasic.UnitTests.VisualBasicParseOptionsTests.CurrentVersionNumber`